### PR TITLE
Remove strings transformations

### PR DESCRIPTION
**Pull request by PiotrBlachnio**

### CODE REVIEW

**Comment by bakshiutkarsha** [comment]
@PiotrBlachnio The Travis CI build is not passing successfully, can you please check that?

**Comment by kelson42** [comment]
@PiotrBlachnio Thank you for your PR. What is the context of it? Is that an attempt to fix https://github.com/openzim/mwoffliner/issues/1162 ?

**Comment by PiotrBlachnio** [comment]
@kelson42 Yes, it should be linked to #1162

**Comment by kelson42** [comment]
@PiotrBlachnio OK, I have linked it. Please test your code and secure the CI works. You can run `npm test` to simulate it.

**Comment by PiotrBlachnio** [comment]
@kelson42 All tests have passed successfully on my local machine except Zimcheck ones because it's not installed.

**Comment by kelson42** [comment]
@PiotrBlachnio You should install zimcheck. Here are binaries for GNU/Linux https://download.openzim.org/release/zim-tools/

**Comment by PiotrBlachnio** [comment]
@kelson42 Code has been updated.

**Comment by kelson42** [comment]
@PiotrBlachnio Thx, can you please rebase on most recent `master` branch?

**Comment by PiotrBlachnio** [comment]
@kelson42 It should be up-to-date now.

File: test/e2e/bm.e2e.test.ts [hunk 1]
```diff
@@ -2,10 +2,9 @@ import test from 'blue-tape';
 import { execute } from '../../src/mwoffliner.lib';
 import execa from 'execa';
 import rimraf from 'rimraf';
-import { zimcheckAvailable, zimcheck } from 'test/util';
+import { zimcheckAvailable, zimcheck } from '../util';
```
Comment (kelson42) [inline test/e2e/bm.e2e.test.ts:5]: I don't think this is necessary or even wishable.

File: test/unit/bootstrap.test.ts [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
-// tslint:disable-next-line: no-var-requires
-require('dotenv').config();
-
-import Redis from 'src/Redis';
+import 'dotenv/config';
+import Redis from '../../src/Redis';
 import test from 'blue-tape';
 import { config } from '../../src/config';
-import { articleDetailXId, redirectsXId, filesToDownloadXPath, populateArticleDetail, populateRedirects, populateFilesToDownload } from 'src/stores';
+import { articleDetailXId, redirectsXId, filesToDownloadXPath, populateArticleDetail, populateRedirects, populateFilesToDownload } from '../../src/stores';
```
Comment (kelson42) [inline test/unit/bootstrap.test.ts:8]: I don't think this is necessary or even wishable.

**Comment by PiotrBlachnio** [comment]
@kelson42 Changes have been implemented.

**Comment by kelson42** [comment]
@PiotrBlachnio Thank you very much for this PR. I'll merge it. Feel free to pick and other one (please put a comment before on the issue so we know you are working on it). Feel free to ask your questions as well.

**Pull request merged**